WatchFaceKotlin/build.gradle

```diff
@@ -42,31 +24,21 @@ buildscript {
 }
 
 plugins {
-    id "com.github.ben-manes.versions" version "0.41.0"
-    id "nl.littlerobots.version-catalog-update" version "0.3.0"
+    alias(libs.plugins.com.diffplug.spotless) apply(false)
+    alias(libs.plugins.com.android.application) apply(false)
 }
 
 allprojects {
-    repositories {
-        google()
-        mavenCentral()
-    }
-}
+    apply plugin: "com.diffplug.spotless"
 
-task clean(type: Delete) {
-    delete rootProject.buildDir
-}
+    spotless {
+        kotlin {
+            target "**/*.kt"
+            targetExclude("$buildDir/**/*.kt")
+            targetExclude("bin/**/*.kt")
 
-tasks.named("dependencyUpdates").configure {
-    rejectVersionIf {
-        (it.candidate.version ==~ /.*-alpha.*/ && !(it.currentVersion ==~ /.*-alpha.*/)) ||
-                (it.candidate.version ==~ /.*-beta.*/ && !(it.currentVersion ==~ /.*-(beta|alpha).*/)) ||
-                (it.candidate.version ==~ /.*1.6.20-M.*/)
+            ktlint("0.41.0")
+            licenseHeaderFile rootProject.file("spotless/copyright.kt")
```

Comment:
Missing, but worth adding if it doesn't trigger a bunch of failures.

```
> java.nio.file.NoSuchFileException: /home/runner/work/wear-os-samples/wear-os-samples/WatchFaceKotlin/spotless/copyright.kt
```

Reply:
Copyright was missing (fixed now), but ktlint check fails as well, commented out it temporarily, but plan to take a look a bit later.